lib/http.js

```diff
@@ -3,7 +3,7 @@ const https = require('https');
 
 class HTTP_Client {
 
-    constructor(timeout = 12000){
+    constructor(timeout = 60000){
```

Comment:
would it make sense to make the timeout equivalent to the worker timeout or is that just asking for trouble?

Reply:
I would be concerned about leaving a request open that long, and I want us to get rid of this entire HTTP client sooner than later anyway.

Reply:
very fair